java/util/src/main/java/com/google/protobuf/util/JsonFormat.java

```diff
@@ -114,7 +114,8 @@ public static Printer printer() {
         /* preservingProtoFieldNames */ false,
         /* omittingInsignificantWhitespace */ false,
         /* printingEnumsAsInts */ false,
-        /* sortingMapKeys */ false);
+        /* sortingMapKeys */ false,
+        /* alwaysWithQuotes */ true);
```

Comment:
Can we name this feature flag "quoteIntegers"?

Reply:
How about `quoteUInt64`? Other integers aren't ever quoted.